### ISSUE TITLE
Add GMO crops

### ISSUE THREAD

**Issue by ThePiachu** [StrangeLoopGames/EcoSuggestions#1304]
I was talking some crop balance recently, how some crops like corn are basically supercrops, and an idea came to my head - what if we introduced GMO crops to the game?

Something for the late game, where you could turn some seeds into GMO seeds with a lab. Those could come in a few varieties even - higher-yield crops, faster growing crops, crops that give you more fibre, etc. You could make some of the recipes a bit cheap but say, make the GMO plants sterile (so you always have to buy fresh seeds from the lab to plant them, like a certain company is doing...), or make them need a lot more fertilisers to maintain the high yield. Basically crops that take extra work and extra care but are really strong in comparison to regular crops.

You could perhaps have some low-tech variation of this as well with "selective breeding" recipe at a farmer's desk or something. Take a good deal of time, a lot of labour and 50 seeds to make 1 good seed of a mid-range GMO that works better than regular crop to basically represent stuff like going from heritage potatoes and crops to the modern potato / crop culture through manual crop selection. You could still gather good seeds from these plants to keep the new strain spreading on farms that's different from wild crops.

**Comment by Engimage** [comment]
I agree with this one.
There could even be like 3 subtypes of the crops.
1. Wild crops. Normal yield, gathering grants wild seeds.
2. Domesticated crops (Produced by a Farmer) - increased yield, less strict requirements for climate. Gathering grants domesticated seeds. These might be called "selected" seeds or alike. It makes sense to process fruits/vegies to find the best seeds among them.
3. GMO crops (some late game profession, probably new) - dramatically increased yield, very broad climate requirements.

And yeah, higher tiers of plants can also require more nutrients making fertilizers a must for them.

**Comment by ThePiachu** [comment]
Hmm, thinking about it, GMO crops would also address an issue I heard from some farmers - they are often the ones that need the most land on a server due to having to grow so much crops. If you had GMO crops with increased yield that would mean they could increase the yield over time without having to have gigantic plots of land, alleviating that issue. Make those plants use up about the same amount of fertiliser per unit of food to drive the fertiliser demand up and things are looking quite interesting.

One issue would be that a lot more recipes would have to use tags - you could get a basic salad from Corn Item, Beet Item, Tomato Item, and then each of those would have their Wild, Domesticated and GMO variant. This would be needed to be able to turn those into their proper seed variants.

**Comment by Engimage** [comment]
I do not think that you should actually change the item itself, and recipes as a result. You can opt to change only the plants and the seeds. GMO plants will only give you more yield but same fruits. However, the way you get the seeds will differ. GMO plants will naturally drop GMO seeds, but to get additional seeds you will still need that profession like the Farmer to get more from the fruits themselves.

Also while having more yield per square, it is important to keep the farmer busy, and keep the usefulness of the tractor.

Thats where we can add controlled re-tilling and calories spent for planting seeds. But that's another suggestion topic.

**Comment by ThePiachu** [comment]
> I do not think that you should actually change the item itself, and recipes as a result.

You would need to change some things if you wanted say, Domesticated Corn to drop you Domesticated Corn Kernels to turn into Domesticated Corn Seeds at a better rate than Wild Corn Kernels to Domesticated Corn Seeds.

----

So the way I'd see something like Corn going:
- Wild Corn - drops 1-2ish Wild Corn Kernels, which you can use directly in a recipe or put back into the ground as seeds. It's a wild variant, so it's basically a glorified grass with a few Kernels. This is a little bit weaker than the current Corn, but you don't need any Farming skill to produce seeds. Consumes very little soil resources.
- Domesticated Corn Seed Recipe - takes 50 Wild Corn Kernels, a good deal of labour and time, produces 1 Domesticated Corn Kernel. Very inefficient, but it reflects years of selective breeding. Accepts Modules for extra efficiency. You can plant the new Kernels into the ground and they will grow Domesticated Corn.
- Domesticated Corn - drops 1 Corncob and 1-3 plant fibres. Can't plant the Corncob directly, but you can eat it. Consumes a moderate amount of soil resources.
- Corncob processing - takes a bit of farming labour, turns 1 Corncob to 2-3ish Domesticated Corn Kernels, accepts Modules for extra efficiency, so the final yield would be like, 4-6 Domesticated Corn Kernels. A bit better than current Corn, but requires a little bit of labour.
- GMO Corn Seed Recipe - Uses the Lab, takes 3-5 Domesticated Corn Kernels, produces 1 GMO Corn Kernel. Fast enough to be able to mass-produce those seeds for the world. Uses Modern Modules and like, Farming 6-7 labour.
- GMO Corn - drops 2-3 Corncobs and 1-3 plant fibres. The kicker is - those are Domesticated Corncobs, meaning you have to keep getting the GMO Corn seeds on the regular. The process is worth it because the yield is 3 times bigger, but requires constant Lab work to keep up. Consumes a lot of soil resources.

----

Something like Wheat could work similar. One difference would be with harvesting it. When you cut down Domesticated Wheat, you get something that represents the whole plant that you have to [thresh](https://en.wikipedia.org/wiki/Threshing) - high-calorie Farming labour that creates Wheat you can plant and also plant fibres. Then the kicker would be that if you have a combine (whether that's the current Tractor or something new) that is done automatically so you want to progress the tech to make it easier. You could also have some T2 machine for threshing, and maybe the pre-threshing wheat would also stack in like hay bales or something? Some stuff that takes a bit of volume, maybe 50-100 to a cube?

**Comment by ThePiachu** [comment]
Or if you'd like the GMO crops to reproduce as well - you'd make them drop the GMO versions of their crops which could be turned into GMO seeds like you do with normal seeds now.

**Comment by D3nnis3n** [comment]
The problem is that, to fit into the style of the game, GMO needs to have some massive technical and moral disadvantages to resemble the fact that they are prohibited on half of the world, giving an incentive to make that decision in Eco as well.

**Comment by ThePiachu** [comment]
Hmm, maybe the GMO crops would come with its own kind of pollutant that would be harmless to them, but would kill any other plants in the area akin to normal ground pollution. This would mean that once you switch to GMOs you can only farm GMOs in that area. Compound it with GMO seeds you need to get from Lab and you have a cycle of dependency mirroring some real-world struggles - going into GMOs makes you dependent on GMO seed producers.

This would probably be the simplest solution until the game has some pesticides or something - GMO plants could be immune to them and thrive from being in pesticide heavy environments, while other plants might only tolerate them so much before dropping their yields or dropping dead.

To simulate the problem of GMO seeds spreading in the wild and causing issues, you could give each GMO crop a chance to produce some wild mutant plant in the vicinity that if left unchecked could over-run the area. It could be some sort of weed crop that doesn't produce much of anything, just choking the rest of the plants out by consuming nutrients? Or just the GMO variant that eats through all the nutrients and devastates the area if left unfertilised.

So yeah, the downsides would probably have to focus on something simulating pesticides, or something about them spreading uncontrollably, as well as the economic side of things. Not sure what else is out there according to [research](https://oxfordre.com/environmentalscience/view/10.1093/acrefore/9780199389414.001.0001/acrefore-9780199389414-e-217). This could then say, synergize well with #1313 where the hydroponic farms would limit the impact on the natural environment due to being confined, etc.

Hmm... GMOs and patents would on one way be interesting, on another it's recreating the dynamic everyone already hates with one group having all the tech power on the server...

**Comment by ThePiachu** [comment]
Once something like radioactive materials get introduced, those might also be used to create GMO seeds (AFAIR irradiating seeds is one of the methods of increasing the mutation chances), which would mean creating them would also have to be done with some extra radiation safety. That could be another drawback to GMO seeds.

**Comment by Engimage** [comment]
@D3nnis3n 
It is fun how you want to translate some weird mass misunderstanding or rumors into the game with your own subjective vision of the matter :)

IRL there is no scientific proof that GMO products are harmful to a human in any way. And surely there is no actual pollution or something alike associated with it.
Yes the products are innaturally modified giving them some new qualities but is is far from being a direct disadvantage.

Its like being a scientist during medieval age - people fear of the unknown. GMO products are not proven harmless (not harmful) but that is enough for the modern society to stick a label on it.
And trust me, all that "organic" or "GMO-free" labels are a complete garbage. There is no way to prove if the product actually contains GMO or not. Hell you can't even tell if the seed you get in the first place has not been a result of cross-pollination with GMO plants getting a trace amounts of modifications.

@ThePiachu 
And GMO products are not a result of a random process such as radiation mutation. GMO is a controlled DNA modification for benefit, while radiation is a randomized process most oftenly leading to negative results.

**Comment by D3nnis3n** [comment]
That is simply untrue, sorry.

**Comment by D3nnis3n** [comment]
This sentiment might come from the reversed ways in how safety regulations work. In the US you need to proof something is not safe, in the EU you need to proof something is safe.

**Comment by Engimage** [comment]
Getting away from the rant, GMO plants might have several modifications and that actually might lead to certain discussions on the server.

For example:
- They can actually suffocate other plants in the area
- They can have a huge self-replenishment rate meaning if they are growing somewhere they can actually act like an aggressive weed
- They can consume a huge amounts of nutrients
> Hence the regulations for each and every modification to be proven safe before being allowed - which does not happen.
^^ This. Some select products were caught being harmful. However this was enough to put a label on all of them while only a tiny portion would actually be really harmful.

**Comment by D3nnis3n** [comment]
Yes, but that's european sentiment. And i want this to be resembled in the game. We want stuff be proven safe, not the other way round. Otherwise we could throw overboard with pretty much any ecological topic. Brasil doesn't think it's any issue to burn down the rainforest. A lot of countries don't think it's any issue that animals are hoarded on small places to deliver great meat for us. (Including my own country!) But whenever we introduce that, we want to actually have the player have a hard time to find a decision, not simply take the optimal route for maximum efficiency in the game. (And yes, we do fail at that somewhat)

**Comment by Engimage** [comment]
I do understand you here. Thats why I am suggesting to add a clear qualities that would actually be the downside of the process. Hence you can't add some sleeper health damage effect to the plant or its products, but you can make the plant do irregular stuff like overconsumption of nutrients or weed behavior

**Comment by D3nnis3n** [comment]
Yes, that's totally fine. I just feel like there needs to be some drawback additionally to morals, obviously in realistic constraints. Cause morals in games don't really work often, because it's a game, lol.

**Comment by ThePiachu** [comment]
> And GMO products are not a result of a random process such as radiation mutation. GMO is a controlled DNA modification for benefit, while radiation is a randomized process most oftenly leading to negative results.

There are many techniques to creating GMOs. One of the older and more crude was causing genetic mutations in plants by exposing them to some forms of radiations (not necessarily radioactive materials, but things like X-rays, etc.). Others include stuff like gene guns (bombarding stuff with DNA on gold particles), using some bacteria to introduce new genes in, etc. All in all, there are a lot of approaches to making GMO foods and introducing new genes to plants that the game could take inspiration from.

Here are some overviews of the related topics:

https://www.youtube.com/watch?v=sH4bi60alZU
https://www.youtube.com/watch?v=7TmcXYp8xu4

From what I understand, the biggest drawbacks / objections to GMOs are:

- Terminator seeds - plants that produce sterile seeds, meaning farmers are in the pockets of seed producers
- Patents - a lot of GMO stuff is patented, and if you get GMO seeds mixed with your non-GMO seeds, you might just be sued and lose all your crops
- Pesticides - making pesticide-resistant crops makes farmers over-use pesticides, which is harmful to the environment
- Built-in pesticides - making plants producing their own pesticides that kill insects feeding on them, which can also kill other insects that are not intended
- Cross-contamination - genes from the GMO plants ending up in non-GMO plants

On the flip side, you also get:
- Plants resistant to pests, infections, fungi, etc.
- Plants that use less farmland, require less fertilisers (which themselves can poison the area if over-used)
- Higher yields
- Adding other beneficial traits (golden rice, etc.)

----

So from that:

- GMOs could be all terminator seeds, requiring the use of some high-end lab to make it. Make the process a bit expensive, maybe even make the lab use a T4 room so not all farmers would be able to afford to make their own GMO crops and you have a good reflection of the divide between haves and have-nots
- You could mimic pests attacking domesticated plants and GMO plants by saying that if there are too many identical plants in a vicinity (like on a large farm) some of them might get attacked by pests. Using pesticides would lower the chance, GMO foods would be resistant to high doses of pesticides. Over-using pesticides however would mean the wild plants in the area would die off and local invertebrates would die off. You could model pesticides like fertilisers mechanically.
- You could also introduce some plant diseases that could similarly decimate mono cultures of crops. These might be a bit punitive though...
- GMOs could generally be bad for the invertebrates in the area
- Use of fertilisers could lead to some new form of ground pollution that perhaps domesticated and GMO plants would be somewhat resistant to, but could affect the world in some other way (maybe polluting the nearby water?)
- Add some sort of penalty to clearing too much land for agricultural use. Maybe some sort of soil degradation or something. Then GMOs could be used to sort of combat that by using less land for the same amount of yield. Alternative would be vertical farms and hydroponics - #1313

Unfortunately, I doubt there is a way to mimic something being an invasive species in ECO since if some plant would grow too well in an area it's not from you'd just see it migrate there too easily due to how the biomes are randomly connected.

**Comment by redwyre** [comment]
> This sentiment might come from the reversed ways in how safety regulations work. In the US you need to proof something is not safe, in the EU you need to proof something is safe. Evidence of most studies in the EU show that GMO is _not safe_ and can at the very least not be proofed safe. Hence the regulations for each and every modification to be proven safe before being allowed - which does not happen.

I don't know where that's coming from, the EU doesn't wholesale ban GMOs (though some specifics strains may be), It's up to member states if they want to allow _farming_ of GMOs (eg German has banned GMO _crops_). I don't see anything stating that the purchase of GMOs in banned anywhere in the EU.

**Comment by karlthepagan** [comment]
> The problem is that, to fit into the style of the game, GMO needs to have some massive technical and moral disadvantages to resemble the fact that they are prohibited

When it comes to the pro/con on GMO I'd be happy with a broad social and scientific representation of the issue. In general the connection between GMOs, pesticides, and fertilizers has more to do with the influence of capitalism than it does with the technical solutions.

Selective breeding is a very important part of human agriculture and has produced both tremendous benefits as well as deadly mistakes (the loss of the Big Mike banana from crop monoculture, the case of the poisonous Lenape potato, or the overuse of pesticides that MIGHT be associated with Roundup Ready Corn).

GMO whether from wild technological experiments ("atomic gardening" produced red grapefruits!) or intentional gene analysis and breeding (such as the winter wheat project at WSU) or gene sequencing (Roundup Ready Corn and drought resistant cops)  is something that deserves a VERY well researched implementation.

I recommend ways that players can be tempted to corner the market using GMOs and industrial synergies. These should have explicit lessons that relate to historic cases but also the opportunity for success or failure as players advance their craft.
